### PR TITLE
Use superfine

### PR DESCRIPTION
**Pull request by sea-grass**
See commit notes for details

### CODE REVIEW

File: docs/index.js [hunk 1]
```diff
@@ -1,132 +1 @@
-(function () {
```
Comment (thegrassisstillgreen) [inline docs/index.js:1]: Did you mean to minify this file?

Comment (thegrassisstillgreen) [reply docs/index.js:1]: I guess so beceause this isn't coming from src

File: src/index.js [hunk 2]
```diff
@@ -1,121 +1,145 @@
 import pool from './pool.js';
-let roundsPlayed = 0;
-let tiesPlayed = 0;
-let result = "new-game";
-let playerOneChoice = "nothing yet";
-let playerTwoChoice = "nothing yet";
-const scores = [0, 0];
+import { h, text, patch } from 'superfine';
```
Comment (thegrassisstillgreen) [inline src/index.js:2]: superfine definitely seems to make this file a bit cleaner. Nice!

**Comment by sea-grass** [comment]
I agree! Now it's looking... super fine 😎

**Pull request merged**